docs_versioned_docs/version-ros2humble/ros/installation/robot.mdx

```diff
@@ -12,12 +12,54 @@ import ComputerNetworkingSetupInstall from "../../components/networking/_compute
 
 ## Operating System (OS) {#operating-system}
 
-ROS 2 Humble uses [Ubuntu 22.04](https://releases.ubuntu.com/jammy/) as its Tier 1 operating system.
-Though other operating systems are supported,
-it is highly recommended to use Ubuntu 22.04. For the robot computer, Ubuntu 22.04 Server should be
-installed. The server version of the OS will not install a graphical user interface (GUI).
+ROS 2 Humble uses Ubuntu 22.04 as its Tier 1 operating system.
+Though other operating systems are supported, it is highly recommended to use Ubuntu 22.04. 
+The preferred method is using the Clearpath Robotics ISO image, which is covered in this section.
+
+:::note
+
+The Clearpath Robotics ISO image only targets x86 computers (`amd64` architecture).
+
+:::
+
+Clearpath provides a lightly customized installation image of Ubuntu 22.04 that automatically has the needed script to setup the robot.
+To install the software on a physical robot through the Clearpath Robotics ISO image, you will first need a USB drive of at least 4 GB
+to create the installation media, an ethernet cable, a monitor, and a keyboard.
+
+1.  Download the [ROS 2 Humble ISO image](https://packages.clearpathrobotics.com/stable/images/1.0.0/).
```

Comment:
Will we update this link each time there is a new release?

Reply:
That is correct.